### PR TITLE
Add Katello 3.10

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: lib/tool_belt/mash_script.rb [hunk 1]
```diff
@@ -25,10 +25,10 @@ def template
 multilib_method = devel
 tag = #{self.tag}
 inherit = True
-strict_keys = False
+strict_keys = True
```
Comment (ehelms) [inline lib/tool_belt/mash_script.rb]: @ekohl Changing this for all based on discussions

**Comment by ehelms** [comment]
@ekohl I also added making procedures command more generic

File: lib/tool_belt/mash_script.rb (hunk 1 above)
Comment (ehelms) [reply lib/tool_belt/mash_script.rb]: This is now a parameter.

**Comment by ehelms** [comment]
Updated @ekohl

File: configs/katello/3.10.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  :hammer-cli-katello:
+    :branch: 0.14.0
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 3.3.5
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :bastion:
+    :branch: 6.1.12
+    :repo: https://github.com/Katello/bastion.git
+  :puppet-pulp:
```
Comment (ekohl) [inline configs/katello/3.10.yaml]: Could you sort these? makes it easier to verify.

I also miss puppet-katello. The procedure omitted it as well.

**Comment by ehelms** [comment]
Updated

File: configs/katello/3.10.yaml [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    :branch: 3.3.5
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :bastion:
+    :branch: 6.1.12
+    :repo: https://github.com/Katello/bastion.git
+  :puppet-candlepin:
+    :branch: 5.2.2
+    :repo: https://github.com/theforeman/puppet-candlepin
+  :puppet-certs:
+    :branch: 4.3.1
```
Comment (ekohl) [inline configs/katello/3.10.yaml]: Why 4.3.1? Did you mean 4.4.1? Now that 4.4.2 is out, let's use that instead :) Or just 4.4-stable.

File: configs/katello/3.10.yaml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+    :repo: https://github.com/theforeman/puppet-certs
+  :puppet-foreman_proxy_content:
+    :branch: 7.2.0
```
Comment (ekohl) [inline configs/katello/3.10.yaml]: 3.9 used 7.3.2

File: configs/katello/3.10.yaml [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+    :repo: https://github.com/theforeman/puppet-foreman_proxy_content
+  :puppet-katello:
+    :branch: 8.2.0
+    :repo: https://github.com/theforeman/puppet-pulp
+  :puppet-pulp:
+    :branch: 5.5.0
```
Comment (ekohl) [inline configs/katello/3.10.yaml]: 3.9 used 5.7.0

File: configs/katello/3.10.yaml [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+    :branch: 0.14.0
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 3.3.5
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :bastion:
+    :branch: 6.1.12
+    :repo: https://github.com/Katello/bastion.git
+  :puppet-candlepin:
+    :branch: 5.2.2
```
Comment (ekohl) [inline configs/katello/3.10.yaml]: 5.2.2 doesn't exist.

File: configs/katello/3.10.yaml [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+    :repo: https://github.com/theforeman/puppet-candlepin
+  :puppet-certs:
+    :branch: 4.3.1
+    :repo: https://github.com/theforeman/puppet-certs
+  :puppet-foreman_proxy_content:
+    :branch: 7.2.0
+    :repo: https://github.com/theforeman/puppet-foreman_proxy_content
+  :puppet-katello:
+    :branch: 8.2.0
+    :repo: https://github.com/theforeman/puppet-pulp
```
Comment (ekohl) [inline configs/katello/3.10.yaml]: puppet-katello?

File: configs/katello/3.10.yaml [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+  :puppet-pulp:
+    :branch: 5.5.0
+    :repo: https://github.com/theforeman/puppet-pulp
+  :puppet-qpid:
+    :branch: 4.3.1
```
Comment (ekohl) [inline configs/katello/3.10.yaml]: 4.3.1 doesn't exist

**Comment by akofink** [comment]
@cfouant you might want to take a look at this as well.

File: configs/katello/3.10.yaml [hunk 9]
```diff
@@ -0,0 +1,86 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.10.1:
```
Comment (akofink) [inline configs/katello/3.10.yaml]: Should be 3.9 something.

Comment (ehelms) [reply configs/katello/3.10.yaml]: Thanks @akofink got it fixed!

**Comment by ehelms** [comment]
Updated

**Pull request merged**